Session.js

```diff
@@ -113,6 +113,33 @@ module.exports = class Session {
     return allCandidates[0];
   }
 
+  /**
+   * conditionalDecide
+   * Decision option to determine decision to take.
+   *
+   * @param  {string} name - name of event
+   * @param  {array} contexts - contexts to choose from
```

Comment:
My understanding is that this should be a superset of a regular decide call? Is that not the case. So shouldn't it also include a decide event name and candidates?